src/API/APIcalls.js

```diff
@@ -33,10 +32,10 @@ const fetcher = {
         },
         body: JSON.stringify(userInfo)
     }
-      const fetchedUser = `https://rancid-tomatillos.herokuapp.com/api/v2/movies/login`
-      const promise = fetch(fetchedUser, int)
+      const fetchedUser = `https://rancid-tomatillos.herokuapp.com/api/v2/login`
+      return fetch(fetchedUser, int)
         .then(request => request.json())
-      return promise
+        .then(data => data)
         .catch(err => console.log('err', err))
     },
```

Comment:
Did we ever figure if this was the better way to run these? Should we change them all to this format?